src/components/IssueQueueItem.vue

```diff
@@ -13,15 +13,20 @@ const props = defineProps<{
 const nonIdFields = ['impact', 'formattedDate', 'title', 'workflowState', 'owner'];
 
 const isUnembargoDateScheduledForLater = computed(
-  () => DateTime.fromISO(props.issue.unembargo_dt).diffNow().milliseconds > 0,
+  () => DateTime.fromISO(props.issue.unembargo_dt).diffNow().milliseconds > 0
 );
 
 const isEmbargoedButNotScheduled = computed(
-  () => props.issue.embargoed && !props.issue.unembargo_dt,
+  () => props.issue.embargoed && !props.issue.unembargo_dt
 );
 
 const isEmbargoed = computed(
-  () => isUnembargoDateScheduledForLater.value || isEmbargoedButNotScheduled.value,
+  () => (isUnembargoDateScheduledForLater.value && props.issue.embargoed)
+    || isEmbargoedButNotScheduled.value
+);
```

Comment:
I am just thinking, why does this logic needs to be so complicated?
`Flaw.embargoed == true` -> embargoed 
`Flaw.embargoed == false` -> unembargoed (public)

Or am I missing something?